minor tweaks to use (what i think is) the preferred properties syntax and naming conventions

ahgittin opened the pull request.

ahgittin committed:
minor tweaks to use (what i think is) the preferred properties syntax and naming conventions

sjcorbett commented:
Alex, thanks for this. I'm inclined to leave this pull request unmerged and to replace the `RavelloPropertiesFromBrooklynProperties` class with the new `LocationPropertiesFromBrooklynProperties` in Brooklyn core instead (as mentioned by @aledsage in emails). Are you happy for me to do this? I'll include add the addition to `RavelloLocation`.

ahgittin commented:
+1 kill the `RavelloProps...` .  note use of `execScript` and `user` (not `run` and `username`) also changed here.  feel free to close this PR if you've got it under control.

also check out `CloudMachineNamer` which i've refactored to be accessible to all locations, if you have to generate a machine or group name.  this makes the name "nice", based on brooklyn app+entity id+name, subject to given length constraints.

ahgittin commented:
no longer relevant
